### ISSUE TITLE
no keyboard navigation

### ISSUE THREAD

**Issue by togakangaroo** [BMSVieira/BVSelect-VanillaJS#8]
Was looking into this to see if I should recommend it to a friend. Being a11y friendly is always a big deal. I noticed that I cannot use this with keyboard navigation. I cannot open a selectbox with a keyboard trigger and I cannot tab to select boxes.

I get this is your open source project and not making any demands, just saying that accessibility is an important aspect of choosing a select control with actual legal implications in the US (for the site that uses it). You might consider either prioritizing these features or clearly stating the limitation at the top of the readme

**Comment by MohsenAlyafei** [comment]
I agree. This would be a necessary feature for compliance purposes.

Hope it gets fixed/added.

> Was looking into this to see if I should recommend it to a friend. Being a11y friendly is always a big deal. I noticed that I cannot use this with keyboard navigation. I cannot open a selectbox with a keyboard trigger and I cannot tab to select boxes.
> 
> I get this is your open source project and not making any demands, just saying that accessibility is an important aspect of choosing a select control with actual legal implications in the US (for the site that uses it). You might consider either prioritizing these features or clearly stating the limitation at the top of the readme

I agree. This would be a necessary feature for compliance purposes.

Hope it gets fixed/added.

**Comment by BMSVieira** [comment]
Hello,

In my country there is no law or legal implication that demand keyboard navigation in selectboxes, that's why it is not included.
It was in my plans to add it down the road, but i dont have much time now.

Anyway, feel free to submit a PR!

Thank You!

**Comment by emrah-boyoglu** [comment]
keyboard navigation should come urgently
